msal/src/main/java/com/microsoft/identity/client/internal/authorities/Authority.java

```diff
@@ -205,16 +228,30 @@ public static boolean isKnownAuthority(Authority authority) {
         //Check if authority host is known to Microsoft
         knownToMicrosoft = AzureActiveDirectory.hasCloudHost(authority.getAuthorityURL());
 
-        return (knownToDeveloper || knownToMicrosoft);
+        final boolean isKnown = (knownToDeveloper || knownToMicrosoft);
+
+        Logger.verbose(
+                TAG + methodName,
+                "Authority is known? [" + isKnown + "]"
```

Comment:
would it make sense to log these 2 booleans separately?

Reply:
Yep - updated